server/controllers/services/email.js

```diff
@@ -183,16 +183,26 @@ export const webhook = (req, res, next) => {
   // If an email is sent to [info|hello|members|admins|organizers]@:collectiveSlug.opencollective.com,
   // we simply forward it to admins who subscribed to that mailinglist (no approval process)
   if (mailinglist === 'admins') {
-    return sendEmailToList(recipient, {
-      subject: email.subject,
-      body,
-      from: email.from,
-    })
-      .then(() => res.send('ok'))
-      .catch(e => {
-        debugWebhook('Error: ', e);
-        next(e);
-      });
+    return models.Collective.findOne({ where: { slug: collectiveSlug } }).then(collective => {
```

Comment:
Rewrite with `async/await` perhaps?

Reply:
No touching to be able to merge now but I'll be more careful about removing promises to switch to async/await when possible in the future